src/main/scala/edu/berkeley/ce/rockslicing/RockSlicer.scala

```diff
@@ -261,15 +266,18 @@ object RockSlicer {
     *
     * @param block1 First input block
     * @param block2 Second input block
+    * @param tolerance Tolerance for difference between compared values. Defaults to
+    *                  EPSILON
     * @return List of processor faces that are shared by the two blocks. Will be
     *         empty if they share no faces
     */
-  def findSharedProcessorFaces(block1: Block, block2: Block): Seq[Face] = {
+  def findSharedProcessorFaces(block1: Block, block2: Block, tolerance: Double = NumericUtils.EPSILON):
+  Seq[Face] = {
```

Comment:
The indentation here is messed up.

Either put the return type on the same line as the rest of the method signature, or double indent it on the next line.